src/api/app/models/kiwi/image/xml_builder.rb

```diff
@@ -13,11 +13,29 @@ def build
 
         doc = update_packages(doc)
         doc = update_repositories(doc)
+        doc = update_description(doc) if @image.description.present?
```

Comment:
I would move the `if` check inside the method as a guard clause. The method should know what to do by itself.

```ruby
def update_description(document)
  document unless @image.description.present?
  #...

end

Reply:
Done.